### ISSUE TITLE
returned invalid playlist

### ISSUE THREAD

**Issue by skiely** [interlark/123tv-iptv#5]
Hello first of all thank you very much for this & ustvgo. Im getting "returned invalid playlist" with the screenshot below when i try alot of channels and im wondering if im doing something wrong?
![Screenshot_20221113-190738](https://user-images.githubusercontent.com/1945927/201539820-7b5d9e78-b8e4-4012-8041-47aea29fdeee.png)
19:05:58 | Refreshing auth key for channel Adult Swim.

19:06:03 | Channel "Adult Swim" returned invalid playlist.

19:06:03 | 192.168.0.220 [13/Nov/2022:19:05:58 +0000] "GET /adult-swim.m3u8 HTTP/1.1" 200 310 "-" "VLC/3.0.18-rc2 LibVLC/3.0.18-rc2"

thanks again..

**Comment by interlark** [comment]
That looks like server sent you invalid playlist, maybe that time they was on maintenance or something. Do you still have this issue?

**Comment by skiely** [comment]
Yes it still happens, it's always happened for me (about 3 or 4 months) I'm just getting round to asking about it now.

**Comment by vortex91** [comment]
I am having this issue on all channels except FOX NEWS

**Comment by skiely** [comment]
> I am having this issue on all channels except FOX NEWS

Both 123tv & ustvgo are down & might be down indefinitely. Fingers crossed they come back 🤞🏻

**Comment by vortex91** [comment]
further info on debugging.  Seems like custom headers are root cause of issue.  Remove any custom headers used and issues goes away on 20 channels.  Client side this is ok on jellyfin as example but using tivimate im getting issue.

meaning same IP behind same router using jellyfin client no problem for 20 channels script detect but using tivimate i get only 1 channel FOX NEWS.  (btw his issue was not present on ustvgo)

A fix could be that scirpt removes any header clients send and uses its own generic headers when requesting stream

**Comment by thepwrtank18** [comment]
123tv works completely fine on the website, but not on here.
